docker-compose.yaml

```diff
@@ -1,13 +1,17 @@
-version: '3.9'
```

Comment:
I suggest we move all dev setup to https://github.com/dev-launchers/platform__dev-env.